src/go/k8s/apis/redpanda/v1alpha1/cluster_types.go

```diff
@@ -258,6 +276,18 @@ type AdminAPITLS struct {
 	RequireClientAuth bool `json:"requireClientAuth,omitempty"`
 }
 
+// PandaproxyAPITLS TLS settings for Pandaproxy
+type PandaproxyAPITLS struct {
```

Comment:
could we share the two structs here?

Reply:
I removed anything TLS from this PR (types, webhook).